<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QMainWindow" name="QMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set simulation options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="IRRIGATION_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>171</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Simulation options</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="MODEL_LB">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="MODE_CB"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="SOILUSEVARFLAG_CB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use yearly maps</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TIME_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>171</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Simulation period</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="FROM_LB">
          <property name="text">
           <string>From the 1th day of year</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="FROM_CB"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TO_LB">
          <property name="text">
           <string>To the last day of year</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="TO_CB"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Please check if timeseries are completed for the selected period.</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
          <property name="backgroundVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RESOLUTION_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>319</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Spatial resolution</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cell dimension</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="CELLSIZE_SB">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="DOMAINEXT_LB">
          <property name="text">
           <string>Set domain extension</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QgsExtentGroupBox" name="DOMAINEXT"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="HYDROLOGY_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>319</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Hydrologycal model</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="EVALAY_LB">
          <property name="text">
           <string>Evaporative layer thickness (m)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TRANSLAY_LB">
          <property name="text">
           <string>Transpirative layer thickness (m)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="TRANSLAY_SB">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="CAPRISE_CB">
          <property name="text">
           <string>Simulate capillary rise</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="MINSLOP_LB">
          <property name="text">
           <string>Minimum admissible slope (%)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="MAXSLOPE_LB">
          <property name="text">
           <string>Maximum admissible slope (%)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="EVALAY_SB">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="MINSLOPE_SB">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="MAXSLOPE_SB">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FOLDER_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>171</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Path and folder</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="1" column="0">
         <widget class="QLabel" name="IRRSTART_LB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QgsFileWidget" name="OUTFOLDER_FW">
          <property name="storageMode">
           <enum>QgsFileWidget::GetDirectory</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="OUTFOLDER_LE">
          <property name="text">
           <string>Select folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="OUTPUT_PG">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>171</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Output</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_5"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
